types/notificationFeed/README.md

```diff
@@ -0,0 +1,62 @@
+# Notification Feed [Draft]
+
+## Schema
+
+The SCHEMA is to be described.
+
+Users get different type of notifications from users. 
+It should be used when you need a user to be notified of some event. This event might even happen outside Near Social. For example
+- You've been mentioned on a widget.
+- You are requested to do something on a website you've given permission to send you notifications
+
+On this notifications, you might even want to have a link to be redirected to the corresponding place, always considering security.
+
+## Example 1
+
+In this case, Alice would create a notification to Bob letting Bob know Alice mentioned Bob in Alice's main page with no message
+
+```json
+{
+  "alice.near": {
+    "index": {
+      "notify": {
+        "key": "bob.near",
+        "value": {
+          "type": "mention",
+          "item": {
+            "type": "social",
+            "path": "alice.near/post/main",
+            "blockHeight": 88260018,
```

Comment:
`Item` shouldn't include the message. Item is more like a pointer. It provides enough information to identify the given item. E.g. for a post it indicates the following:
- type = "social" - means it belongs to social DB
- path = "alice.near/post/main" - absolute path the item, in this case for `alice.near`
- blockHeight = 88260018 - the block height when the information at the path above is written, if omitted it means the same as the notification (for example when you post smth, you don't have a block height until it's saved).